todo_app/app.py

```diff
@@ -13,6 +13,20 @@
 toDoListId = app.config['TRELLO_TO_DO_LIST_ID']
 doneListId = app.config['TRELLO_DONE_LIST_ID']
 
+def getCards():
```

Comment:
This function is a nice way to extract that logic, but it's not using a `class` to represent the cards, you're still using the dictionaries created by the response's json directly.

The idea here is to define a class with all of the relevant fields (id, status, title etc.):
```python
class Item():
    def __init__(self, id, ...)
        self.id = id
        ...
```

And then use that class in our code rather than Trello's response directly, e.g. instead of 
```python
        for card in cards:
            card.update({"listName": key})
            cardList.append(card)
```
it might look more like
```python
        for card in cards:
            new_card = Card(card['id'], ... )
            cardList.append(new_card)
```

The aim here is to reduce our dependency on Trello throughout the code, this gives us more control over the cards that are being manipulated/accessed elsewhere in our system